packages/core/src/new-api/internal/execution/execution-strategy.ts

```diff
@@ -43,17 +44,19 @@ export class BasicExecutionStrategy
   }): AsyncGenerator<
     OnchainInteractionMessage,
     JournalableMessage,
-    OnchainResultMessage | null
+    OnchainDeployContractSuccessMessage | null
   > {
     const result = yield {
       type: "onchain-action",
       subtype: "deploy-contract",
+      futureId: deploymentExecutionState.id,
+      transactionId: 1,
       contractName: deploymentExecutionState.contractName,
       value: deploymentExecutionState.value.toString(),
       args: deploymentExecutionState.constructorArgs,
       // TODO: the possibily of undefined for `from` needs
       // a working resolution
-      from: deploymentExecutionState.from ?? "n/a",
+      from: deploymentExecutionState.from,
```

Comment:
`from` state can no longer be undefined on `deploymentExecutionState`. This is cleanup in passing that was missed in an earlier PR.